src/controlbox/connector_discovery_facade.py

```diff
@@ -71,7 +71,7 @@ def _open(self):
         If the connection raises a connection error, it is logged, but not raised
         :return: True if the the connector was tried - the connector was not connected and available
         """
-        connector: Connector = self.connector
+        connector = self.connector
```

Comment:
ah, I didn't realize that. I was installing on python 3.5.2 and this was the only line that needed to be changed to make it work. Given that it is a trivial annotation, do we need it?